cores/esp8266/Esp.cpp

```diff
@@ -171,6 +172,11 @@ uint32_t EspClass::getFreeHeap(void)
     return system_get_free_heap_size();
```

Comment:
Are we sure that this sdk api is the same as:
```
uint32_t fh = ummHeapInfo.freeBlocks * block_size;
...
*free = fh;
```
used in getHeapStats()? After a quick look I can't tell how the integration works between that sdk api and umm.